### PR TITLE
Forward rtcp sender packet event

### PR DESCRIPTION
**Pull request by Karolk99**

### CODE REVIEW

**Review by mat-hek — APPROVED** [review]

**Review by mickel8 — CHANGES_REQUESTED** [review]
Can we update RTCP section in SessionBin docs?

File: lib/membrane/rtp/session_bin.ex [hunk 1]
```diff
@@ -59,6 +59,8 @@ defmodule Membrane.RTP.SessionBin do
   receiver report packets will be sent back through `rtcp_receiver_output` with the same id as `rtp_input` for the RTP stream.
 
   RTCP for outbound stream is not yet supported. # But will be :)
+
+  RTCP sender reports are dispatched as `Membrane.RTCP.SenderReportPacket` events on session bin `:output` pads
```
[the remark this answers is not in the document]
Comment (Karolk99) [reply lib/membrane/rtp/session_bin.ex]: only inboud RTCP sender packets

File: lib/membrane/rtp/session_bin.ex [hunk 2]
```diff
@@ -58,6 +58,10 @@ defmodule Membrane.RTP.SessionBin do
   RTCP packets for inbound stream can be provided either in-band or via a separate `rtp_input` pad instance. Corresponding
   receiver report packets will be sent back through `rtcp_receiver_output` with the same id as `rtp_input` for the RTP stream.
 
+  RTCP sender reports from inbound streams are parsed and then dispatched as `Membrane.RTCP.SenderReportPacket` events on session bin `:output` pads.
+  On each `Pad.ref(:output, ssrc)`, only RTCP sender reports with the corresponding SSRC will be sent.
+  These reports can be used, for instance, to synchronize different rtp streams.
+
   RTCP for outbound stream is not yet supported. # But will be :)
```
Comment (mickel8) [inline lib/membrane/rtp/session_bin.ex:65]: This is outdated too :eyes:

**Review by mickel8 — CHANGES_REQUESTED** [review]

File: lib/membrane/rtp/session_bin.ex (hunk 2 above)
Comment (Karolk99) [reply lib/membrane/rtp/session_bin.ex:65]: I will check it once again.

File: lib/membrane/rtp/session_bin.ex [hunk 3]
```diff
@@ -58,7 +58,12 @@ defmodule Membrane.RTP.SessionBin do
   RTCP packets for inbound stream can be provided either in-band or via a separate `rtp_input` pad instance. Corresponding
   receiver report packets will be sent back through `rtcp_receiver_output` with the same id as `rtp_input` for the RTP stream.
 
-  RTCP for outbound stream is not yet supported. # But will be :)
+  RTCP sender reports from inbound streams are parsed and then dispatched as `Membrane.RTCP.SenderReportPacket` events on session bin `:output` pads.
+  On each `Pad.ref(:output, ssrc)`, only RTCP sender reports with the corresponding SSRC will be sent.
+  These reports can be used, for instance, to synchronize different rtp streams.
+
+  RTCP packets for the outbound stream are available `rtcp_sender_output` pad.
+  RTCP packets will only be produced if rtcp_receiver_report_interval option is set to a value other than nil.
```
Comment (mickel8) [inline lib/membrane/rtp/session_bin.ex:66]:
```suggestion
  RTCP packets for the outbound stream are available on `rtcp_sender_output` pad.
  RTCP packets will only be produced if rtcp_sender_report_interval option is set to a value other than nil.
```

**Review by mickel8 — APPROVED** [review]

**Review by mat-hek — APPROVED** [review]

**Pull request merged**